### ISSUE TITLE
Introduction should describe evolution of guidelines

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/silver#100]
From https://www.w3.org/2002/09/wbs/94845/Silver-ED-21-01-2020/results#xq3

Accessibility guidelines are based on two factors: the state of technology and the state of understanding of disability. Both change. I see this theme throughout the introduction, but not explicitly. We also learn how to implement guidelines.

Personally, I think that what's in the Silver introduction at this point should directly address the issues that were at the heart of why the Silver Task was first impaneled. Why did Silver get started? What about the Web Content Accessibility Guidelines version 2.x (WCAG 2.x) was so off that significant segments of the accessibility community demanded a change? Whatever the answers are to that question need to be the focus of what the Silver First Public Working Draft (FPWD) needs to address. Don't be afraid to identify weaknesses in WCAG 2.x directly, instead of burying those issues in linked resources referenced from the FPWD. The fine grain details around conformance, scoring, structure, etc. can com later, as long as the original purpose for forming a Silver Task force is directly addressed in the First Public Working Draft (FPWD). Use the FPWD process as way to crowd source ideas that will help fill in some of the missing pieces that we know have to be built out. Get the community's buy-in about what needs to change in the next generation of the standard before trying to build a proof of concept for that change.

It would be helpful to include (even as placeholder) a section similar to WCAG 2.1’s “0.5 Comparison with WCAG 2.0”, which helps the reader understand what and how this is different than what came before.

Reading the Inroduction, I went back to the Silver Research Summary slides which feature so prominently. I notice that the Usability Problem Statement #3 Ambiguity in interpreting the success criteria, the next slide has only one answer that may address this problem somewhat: "Silver needs more updated examples across newer technologies". The intentions (expanded scope, more technologies, more disabilities etc) and the aim to have graded conformance rating approach, include user testing etc. indicate strongly that the result will be *more complex* even we manage to provide a better user interface (beginners ramp, searching, filtering, organisational role). I fear that the ambiguity and in turn, the difficulty of a consistent conformance rating is going to increase strongly due to that widening of scope and the more complex rating approach. I have no solution, but I just think we probably can't have it both ways (encompassing many more dimensions AND at the same time making it easy to understand and use).

**Comment by jspellman** [comment]
done in FPWD
